ofborg/src/tasks/build.rs

```diff
@@ -341,6 +338,22 @@ impl notifyworker::SimpleNotifyWorker for BuildWorker {
             .map(|(attr, _)| attr)
             .collect();
 
+        info!("Checking for stdenv rebuild");
+        let in_cache = self
+            .nix
+            .safely_query_cache_for_attr(refpath.as_ref(), buildfile, "stdenv".to_string())
```

Comment:
Nit, but I like seeing `String::from()` over `to_string()` for static strings like this.

```suggestion
            .safely_query_cache_for_attr(refpath.as_ref(), buildfile, String::from("stdenv"))
```